x-pack/legacy/plugins/code/server/routes/search.ts

```diff
@@ -89,6 +89,8 @@ export function documentSearchRoute(router: CodeServerRouter, log: Logger) {
       let scope: string[] = [];
       if (typeof repoScope === 'string') {
         scope = repoScope.split(',');
+      } else if (Array.isArray(repoScope)) {
```

Comment:
I think the `join` in line 29 of the file above echoes with the logic here in line 91, do you need to make some update there as well?

Reply:
do you still need the 2 lines above since you have removed the `repoScope` join on line 28 of the file above?

Reply:
probably not